src/Illuminate/Database/Schema/Grammars/Grammar.php

```diff
@@ -320,13 +320,12 @@ public function wrapTable($table)
      * Wrap a value in keyword identifiers.
      *
      * @param  \Illuminate\Support\Fluent|\Illuminate\Contracts\Database\Query\Expression|string  $value
-     * @param  bool  $prefixAlias
      * @return string
      */
-    public function wrap($value, $prefixAlias = false)
+    public function wrap($value)
     {
         return parent::wrap(
-            $value instanceof Fluent ? $value->name : $value, $prefixAlias
+            $value instanceof Fluent ? $value->name : $value,
```

Comment:
This might be considered a BC, however, `$prefixAlias` is not used in this method and the parent method only accepts one argument I figured it was safe to remove.

Reply:
You can't be sure that this is not used by any Laravel package. As the Grammar functions are used extensively by any package extending Eloquent.